apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/BeneficiaryTransformerTest.java

```diff
@@ -46,13 +56,58 @@ public void transformSampleARecord() {
    */
   @Test
   public void transformSampleARecordWithIdentifiers() {
+
     Beneficiary beneficiary = loadSampleABeneficiary();
 
     Patient patient =
         BeneficiaryTransformer.transform(
             new MetricRegistry(), beneficiary, IncludeIdentifiersMode.INCLUDE_HICNS_AND_MBIS);
+
     assertMatches(beneficiary, patient);
-    Assert.assertEquals(7, patient.getIdentifier().size());
+
+    // Verify patient has 6 identifiers.
+    Assert.assertEquals(6, patient.getIdentifier().size());
+
+    // Verify patient identifiers and values match.
+    assertValuesInPatientIdentifiers(
```

Comment:
If I understand this test correctly, we're asserting that when we have the INCLUDE_IDENTIFIERS flag, we're still returning the unhashed hicn, correct?

Reply:
That's correct.